source/Calamari/Deployment/Journal/DeployedPackage.cs

```diff
@@ -54,6 +54,11 @@ public static IEnumerable<DeployedPackage> GetDeployedPackages(RunningDeployment
 
             foreach (var packageReferenceName in variables.GetIndexes(SpecialVariables.Packages.PackageCollection))
             {
+                if (string.IsNullOrEmpty(packageReferenceName) && variables.IsSet(SpecialVariables.Package.NuGetPackageId))
```

Comment:
I don't understand this line?  Why are we checking `Octopus.Action.Package.NuGetPackageId` is set?

Reply:
This is to check if the code above has already handled the default package. I'm thinking it makes sense to allow calamari to be backwards compatible with the old variable location.